docs/providers/patreon.md

```diff
@@ -0,0 +1,40 @@
+---
+id: patreon
+title: Patreon
+---
+
+## Documentation
+
+https://docs.patreon.com/#apiv2-oauth
+
+## Configuration
+
+:::tip
+  Create a API v2 client on [Patreon Platform](https://www.patreon.com/portal/registration/register-clients)
+:::
+
+## Options
+
+The **Patreon Provider** comes with a set of default options:
+
+- [Patreon Provider options](https://github.com/nextauthjs/next-auth/blob/main/src/providers/patreon.js)
```

Comment:
Let's keep an eye on this. If you end up converting the file to TypeScript, the extension has to change to `.ts`